examples/Std/OptionAndResult.dfy

```diff
@@ -97,32 +97,19 @@ module Demo {
     }
   }
 
-  method TestMyFilesystem() {
+  method TestMyFilesystem() returns (r: Result<()>) {
     var fs := new MyFilesystem();
-    // Note: these verbose "outcome.Failure?" patterns will soon
-    // not be needed any more, see https://github.com/dafny-lang/rfcs/pull/1
-    var outcome: Result<()> := fs.CreateFile("test.txt");
-    if outcome.Failure? {
-      print outcome.error, "\n";
-      return;
-    }
-    outcome := fs.WriteFile("test.txt", "Test dummy file");
-    if outcome.Failure? {
-      print outcome.error, "\n";
-      return;
-    }
-    var fileResult: Result<string> := fs.ReadFile("test.txt");
-    if outcome.Failure? {
-      print fileResult.error, "\n";
-      return;
-    }
-    if fileResult.Success? {
-      print fileResult.value, "\n";
-    }
+    var _ :- fs.CreateFile("test.txt");
+    var _ :- fs.WriteFile("test.txt", "Test dummy file");
```

Comment:
When I implemented `:-`, I intended to support statements starting with `:-`, so that you could drop the `var _ :- ` part in the above code. Did you try that? Does it work? Or do you get an error saying that there must not be an `Extract` method? In any case, I think there should still be a TODO note here